generative_ai/gemini_all_modalities.py

```diff
@@ -24,6 +24,7 @@ def analyze_all_modalities(project_id: str) -> str:
 
     vertexai.init(project=project_id, location="us-central1")
 
+    # Check latest AI models at https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/inference
```

Comment:
No need to add this link. The page where the sample is shown should have the model info.